projects/1.16/assets/more-jellyfish/more_jellyfish/lang/zh_cn.json

```
@@ -0,0 +1,104 @@
+  "block.more_jellyfish.deep_coralstone_slab": "深海珊瑚石台阶",
+  "block.more_jellyfish.deep_coralstone_vertical_slab": "深海珊瑚石竖台阶",
+
+  "block.more_jellyfish.polished_deep_coralstone": "磨制深海珊瑚石",
+  "block.more_jellyfish.polished_deep_coralstone_stairs": "磨制深海珊瑚石楼梯",
+  "block.more_jellyfish.polished_deep_coralstone_slab": "磨制深海珊瑚石台阶",
+  "block.more_jellyfish.polished_deep_coralstone_vertical_slab": "磨制深海珊瑚石竖台阶",
+
+  "block.more_jellyfish.deep_algae_grass": "深海海藻草方块",
+
+
+  "block.more_jellyfish.gelatinous_milk_block": "胶质奶块",
+
+  "biome.more_jellyfish.rock_bottom": "无底洞",
+  "biome.more_jellyfish.kelp_forest": "海藻森林",
+
+  "advancements.more_jellyfish.rock_bottom.title": "无底洞",
+  "advancements.more_jellyfish.rock_bottom.description": "进入无底洞",
+
+  "advancements.more_jellyfish.kelp_forest.title": "赞美神奇海螺！",
+  "advancements.more_jellyfish.kelp_forest.description": "进入海藻森林",
+
+  "item.more_jellyfish.diamond_shard": "钻石碎片",
+  "item.more_jellyfish.diamond_fragment": "钻石碎块",
+  "item.more_jellyfish.emerald_fragment": "绿宝石碎块",
+  "item.more_jellyfish.sponge_chunk": "海绵碎块",
+  "item.more_jellyfish.gelatinous_milk": "胶质奶",
+  "item.more_jellyfish.honey_ball": "蜂蜜球",
+  "item.more_jellyfish.netherite_chunk": "下界合金碎块",
+
+  "item.more_jellyfish.diamond_jellyfish": "钻石水母",
+  "item.more_jellyfish.emerald_jellyfish": "绿宝石水母",
+  "item.more_jellyfish.iron_jellyfish": "铁水母",
+  "item.more_jellyfish.gold_jellyfish": "金水母",
+  "item.more_jellyfish.coal_jellyfish": "煤炭水母",
+  "item.more_jellyfish.redstone_jellyfish": "红石水母",
+  "item.more_jellyfish.lapis_lazuli_jellyfish": "青金石水母",
+  "item.more_jellyfish.slime_jellyfish": "史莱姆水母",
+  "item.more_jellyfish.quartz_jellyfish": "石英水母",
+  "item.more_jellyfish.sponge_jellyfish": "海绵水母",
+  "item.more_jellyfish.glowstone_jellyfish": "萤石水母",
+  "item.more_jellyfish.red_fungal_jellyfish": "红色菌水母",
+  "item.more_jellyfish.brown_fungal_jellyfish": "棕色菌水母",
+  "item.more_jellyfish.warped_fungal_jellyfish": "诡异菌水母",
+  "item.more_jellyfish.crimson_fungal_jellyfish": "绯红菌水母",
+  "item.more_jellyfish.creeper_jellyfish": "苦力怕水母",
+  "item.more_jellyfish.guardian_jellyfish": "守护者水母",
+  "item.more_jellyfish.skellyfish": "骷髅水母",
+  "item.more_jellyfish.honey_jellyfish": "蜂蜜水母",
+  "item.more_jellyfish.clay_jellyfish": "粘土水母",
+  "item.more_jellyfish.cow_jellyfish": "奶牛水母",
+  "item.more_jellyfish.netherite_jellyfish": "下界合金水母",
+
+  "entity.more_jellyfish.diamond_jellyfish": "钻石水母",
+  "entity.more_jellyfish.emerald_jellyfish": "绿宝石水母",
+  "entity.more_jellyfish.iron_jellyfish": "铁水母",
+  "entity.more_jellyfish.gold_jellyfish": "金水母",
+  "entity.more_jellyfish.coal_jellyfish": "煤炭水母",
+  "entity.more_jellyfish.redstone_jellyfish": "红石水母",
+  "entity.more_jellyfish.lapis_lazuli_jellyfish": "青金石水母",
+  "entity.more_jellyfish.slime_jellyfish": "史莱姆水母",
+  "entity.more_jellyfish.quartz_jellyfish": "石英水母",
+  "entity.more_jellyfish.sponge_jellyfish": "海绵水母",
+  "entity.more_jellyfish.glowstone_jellyfish": "萤石水母",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "entity.more_jellyfish.glowstone_jellyfish": "荧石水母",
```

---

已修正